Database.md

```diff
@@ -205,6 +215,76 @@ WHERE
 	AND name   		= ?
 ```
 
+### Share/Unshare a file
+
+```sql
+-- - This query should receive owner_uuid, other_user_uuid, file_to_share_uuid
+-- - First check if session user is owner of the file
+SELECT uuid
+FROM files
+WHERE
+	owner_uuid = ?
+LIMIT 1
+
+-- - SHARE
+-- - Then insert the file if we are the owners
+INSERT INTO shared_files (uuid, file_uuid, user_uuid)
+VALUES (?, ?, ?)
+
+-- - UNSHARE
+-- - Then insert the file if we are the owners
+DELETE FROM shared_files
+WHERE (
+    file_uuid = ?
+    AND user_uuid = ?
+)
+```
+
+### Can I read
+
+```sql
+-- - Query used to verify if current account can read file
+-- - If result is not zero, we can delete
+SELECT COUNT(*)
```

Comment:
This query should be replaced with the stored procedure to recursively check if a parent folder was shared with the user, right?

Reply:
https://github.com/hawks-atlanta/docs/pull/22#pullrequestreview-1605948948